### DISCUSSION TITLE
Trigger Clip via MQTT

### DISCUSSION THREAD

**Discussion by herostrat** [blakeblackshear/frigate#1008] [Ask A Question]
I cannot seem to find a way to get frigate to start/save a clip via MQTT/HTTP.

I know you can enable/disable clips and snapshots, but than it would also need a detection to start recording.

**Comment by dewgenenny** [comment]
+1 on wanting this - Frigate is super good for detecting and recording, however in some cases it would be nice to trigger a start recording using other sensors, for example in my case, I have open/close binary sensors on my external doors, so I would like to trigger recording based on them opening.... 

@blakeblackshear if you could point me roughly in the right direction of where this is handled within the code, I could have a go at creating a pull request for it...

**Comment by herostrat** [comment]
Hey, here is the issue that tracks the progress

https://github.com/blakeblackshear/frigate/issues/1063
